query/src/main/scala/filodb/query/exec/MultiSchemaPartitionsExec.scala

```diff
@@ -42,13 +42,15 @@ final case class MultiSchemaPartitionsExec(id: String,
 
   var finalPlan: ExecPlan = _
 
-  private def finalizePlan(source: ChunkSource): ExecPlan = {
+  private def finalizePlan(source: ChunkSource, span: kamon.trace.Span): ExecPlan = {
+    span.mark("filtered-partition-scan")
     val partMethod = FilteredPartitionScan(ShardSplit(shard), filters)
+    span.mark("lookup-partitions")
     val lookupRes = source.lookupPartitions(dataset, partMethod, chunkMethod)
 
     // Find the schema if one wasn't supplied
     val schemas = source.schemas(dataset).get
-
+    span.mark("finalize-plan")
```

Comment:
Move this to line 50 and call it as "lookup-partitions-done"

Reply:
Done